gtsfm/configs/deep_front_end.yaml

```diff
@@ -8,16 +8,20 @@ SceneOptimizer:
   feature_extractor:
     _target_: gtsfm.feature_extractor.FeatureExtractor
     detector_descriptor:
-      _target_: gtsfm.frontend.detector_descriptor.superpoint.SuperPointDetectorDescriptor
+      _target_: gtsfm.frontend.detector_descriptor.cacher.detector_descriptor_cacher.DetectorDescriptorCacher
```

Comment:
cache or DB? i'm still unsure

Reply:
I am too. Lets see what others think about this.

cc @dellaert  @adisingh50

Reply:
I think cache is growing on me. It describes the functionality better perhaps.

Reply:
Did that in the files